README.md

```diff
@@ -8,3 +8,7 @@ More information about Landlock can be found in the [official website](https://l
 
 This Rust crate provides a safe abstraction for the Landlock system calls along with some helpers.
 See the [Rust Landlock API documentation](https://landlock.io/rust-landlock/landlock/).
+
+## Minimum Supported Rust Version (MSRV)
+
+Changing the MSRV is considered a breaking change and as such will result in a minor version bump. The current MSRV is 1.63.
```

Comment:
I assumed you would consider changing the MSRV a breaking change.

Reply:
I found that we can specify this version (just after the `edition` entry) in Cargo.toml: https://doc.rust-lang.org/nightly/cargo/reference/manifest.html#the-rust-version-field

With this, the readme changes are not necessary.

Reply:
I considered adding it, but given thay this key is only available from rust 1.56, it is effectively useless until we bump the MSRV to a later version.

Reply:
Indeed, we should keep the readme change, but the `rust-version` config is still useful for Rust >= 1.56, and it will display a warning because of unknown Cargo config key for old versions, which gives a hint too.

Please also only add this paragraph to the readme:
> Minimum Supported Rust Version (MSRV): 1.63

Reply:
Right, 1.56 < 1.63 :facepalm:

Reply:
Done